### PR TITLE
{numlib}[iccifort/2020.1.217] CGAL v4.14.3

### PR DESCRIPTION
**Pull request by bartoldeman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by bartoldeman** [comment]
Test report by @bartoldeman
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
build-node.computecanada.ca - Linux centos linux 7.8.2003, Intel Xeon Processor (Skylake, IBRS), Python 3.7.7
See https://gist.github.com/da550d48deb09eeafa2cc06552f05b60 for a full test report.

File: easybuild/easyconfigs/c/CGAL/CGAL-4.14.3-iccifort-2020.1.217.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name = 'CGAL'
+version = '4.14.3'
+
+homepage = 'https://www.cgal.org/'
+description = """The goal of the CGAL Open Source Project is to provide easy access to efficient 
+ and reliable geometric algorithms in the form of a C++ library."""
+
+toolchain = {'name': 'iccifort', 'version': '2020.1.217'}
```
Comment (smoors) [inline easybuild/easyconfigs/c/CGAL/CGAL-4.14.3-iccifort-2020.1.217.eb:8]: @bartoldeman Boost depends on the impi toolchain, so it's probably better to use impi for CGAL as well.

File: easybuild/easyconfigs/c/CGAL/CGAL-4.14.3-iccifort-2020.1.217.eb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+toolchainopts = {'strict': True}
+
+source_urls = ['https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-%(version)s']
+sources = [SOURCE_TAR_XZ]
+checksums = ['5bafe7abe8435beca17a1082062d363368ec1e3f0d6581bb0da8b010fb389fe4']
+
+dependencies = [
+    ('Boost', '1.72.0'),
+]
```
Comment (smoors) [inline easybuild/easyconfigs/c/CGAL/CGAL-4.14.3-iccifort-2020.1.217.eb:17]: @bartoldeman where did all the dependencies of the previous version go? they are not needed anymore?

**Comment by bartoldeman** [comment]
superseded by  #11366

**Pull request closed without merging**